specification/support/resource-manager/Microsoft.Support/stable/2024-04-01/examples/ListSupportTicketsBySubscription.json

```diff
@@ -0,0 +1,105 @@
+              "secondaryConsent": [
+                {
+                  "userConsent": "Yes",
+                  "type": "VirtualMachine"
+                }
+              ],
+              "contactDetails": {
+                "firstName": "abc",
+                "lastName": "xyz",
+                "preferredContactMethod": "email",
+                "primaryEmailAddress": "abc@contoso.com",
+                "preferredTimeZone": "Pacific Standard Time",
+                "country": "usa",
+                "preferredSupportLanguage": "en-US"
+              },
+              "technicalTicketDetails": {
+                "resourceId": "/subscriptions/subid/resourceGroups/test/providers/Microsoft.Compute/virtualMachines/testserver"
+              },
+              "serviceLevelAgreement": {
+                "startTime": "2020-03-20T21:36:18Z",
+                "expirationTime": "2020-03-21T17:36:18Z",
+                "slaMinutes": 240
+              },
+              "supportEngineer": {
+                "emailAddress": "xyz@contoso.com"
+              },
+              "supportPlanType": "Premier",
+              "supportPlanDisplayName": "Premier",
+              "title": "my title",
+              "serviceId": "/providers/Microsoft.Support/services/virtual_machine_service_guid",
+              "serviceDisplayName": "Virtual Machine running Linux",
+              "status": "Open",
+              "createdDate": "2020-03-20T21:36:18Z",
+              "modifiedDate": "2020-03-20T21:36:23Z"
+            }
+          },
+          {
+            "id": "/subscriptions/subid/providers/Microsoft.Support/supportTickets/testticket2",
+            "name": "testticket2",
+            "type": "Microsoft.Support/supportTickets",
+            "properties": {
+              "supportTicketId": "118032014183771",
+              "description": "This is a test - please ignore",
+              "problemClassificationId": "/providers/Microsoft.Support/services/subscription_management_service_guid/problemClassifications/problemClassification_guid",
+              "problemClassificationDisplayName": "Add or Edit VAT, TAX ID, or PO Number",
+              "severity": "minimal",
+              "require24X7Response": false,
```

Comment:
>require24X7Response [](http://example.com/codeflow?start=15&length=19)

Missing required property: advancedDiagnosticConsent
Url: Microsoft.Support/stable/2024-04-01/support.json#L2421:39
ExampleUrl: stable/2024-04-01/examples/ListSupportTicketsBySubscription.json

Reply:
(I noticed this by reviewing ModelValidation errors.) Please fix modelVAlidation errors!

Reply:
Taken care of.